implementations/elixir-jsv/lib/bowtie_jsv.ex

```diff
@@ -0,0 +1,92 @@
+defmodule BowtieJSV do
+  @dialects [
+    "http://json-schema.org/draft-07/schema#",
+    "https://json-schema.org/draft/2020-12/schema"
+  ]
+
+  def main(_argv) do
+    debug("starting")
+    loop_lines(%{})
+    debug("finished")
+  end
+
+  def debug(msg) do
+    IO.puts(:stderr, msg)
+  end
+
+  def debug_value(value, label) do
+    IO.inspect(:stderr, value, label: label)
+  end
+
+  defp loop_lines(state) do
+    debug("loop")
+
+    case IO.read(:stdio, :line) do
+      :eof ->
+        debug("EOF received")
+
+      line ->
+        {:reply, resp, state} = handle_raw_command(line, state)
+        IO.puts(JSON.encode!(resp))
+        loop_lines(state)
+    end
+  end
+
+  defp handle_raw_command(json, state) do
+    with {:ok, cmd} <- JSON.decode(json) do
+      handle_command(cmd, state)
+    end
+  end
+
+  defp handle_command(%{"cmd" => "start", "version" => 1}, state) do
+    start_response = %{
+      version: 1,
+      implementation: %{
+        language: "elixir",
+        name: "jsv",
+        source: "https://github.com/lud/jsv",
+        homepage: "https://github.com/lud/jsv",
```

Comment:
Including the version of your library here (via some Elixir-specific way to lookup what version of your library has been installed) is nice and will make that show up in the UI eventually.

Reply:
Done!